resources/services/s3/buckets.go

```diff
@@ -529,6 +529,9 @@ func resolveS3BucketsAttributes(ctx context.Context, meta schema.ClientMeta, res
 
 	output, err := mgr.GetBucketRegion(ctx, *resource.Name)
 	if err != nil {
+		if c.IsNotFoundError(err) {
```

Comment:
When we `wrap` the error does that break signature so that [this](https://github.com/cloudquery/cq-provider-aws/blob/702d5d0b35fb32e9508271d58fce99532e8b1cdf/client/helpers.go#L289-L292) will always return `false`?